components/dash-core-components/src/fragments/Graph.react.js

```diff
@@ -170,6 +172,9 @@ class PlotlyGraph extends Component {
             frames: figure.frames,
             config: configClone,
         }).then(() => {
+            // clear typesetMath
+            configClone.typesetMath = undefined;
```

Comment:
Is it necessary to clear this? It's already a clone of the input config.

Reply:
Addressed in https://github.com/plotly/dash/pull/1949/commits/0ee422d5aae317c609b511eec8a6625b8e5e59ea.